modules/backend/behaviors/ListController.php

```diff
@@ -558,14 +565,15 @@ public static function extendListColumns($callback)
     {
         $calledClass = self::getCalledExtensionClass();
         Event::listen('backend.list.extendColumns', function ($widget) use ($calledClass, $callback) {
+            /** @var \Backend\Widgets\Lists $widget */
```

Comment:
@bennothommo did we decide to allow these inline comments? I'm not sure why they're worth having?

Reply:
@LukeTowers I think the intention is to show the expected variable returned from methods which may return several types of classes, like the `makeWidget` method does. It's mainly for providing autocompletion through these types of methods. I don't mind keeping them, but I'd hope their use is very minimal.

Reply:
hmmm, not really a fan. It's pretty clear from the event itself what widget is being returned. @jacobdekeizer thoughts?

Reply:
@LukeTowers @bennotjommo It's indeed for autocompletion. It also shows the developer witch widget is returned. We could also change it to `function (\Backend\Widget\Lists $widget) ...`. Is that allowed?

Reply:
@jacobdekeizer hmm, while that would prevent extensibility in terms of being able to inject any other widget in there instead that seems unlikely to be required and I do prefer that in terms of code clarity so I would say go ahead and give it a try and verify that it still works properly.

Reply:
@LukeTowers I've changed it and verified that it still works.